### PR TITLE
[YUNIKORN-1968] Update "Run NVIDIA GPU Jobs in workload"

### PR DESCRIPTION
**Pull request by KatLantyss**
### What is this PR for?
Update "Run NVIDIA GPU Jobs in workload"

### What type of PR is it?
* [ ] - Bug Fix
* [ ] - Improvement
* [ ] - Feature
* [x] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### CODE REVIEW

File: docs/user_guide/workloads/run_nvidia.md [hunk 1]
```diff
@@ -26,30 +26,16 @@ under the License.
...
 ## Yunikorn with NVIDIA GPUs
-This guide gives an overview of how to set up NVIDIA Device Plugin which enable user to run GPUs with Yunikorn, for more details please check [**Kubernetes with GPUs**](https://docs.nvidia.com/datacenter/cloud-native/kubernetes/install-k8s.html#option-2-installing-kubernetes-using-kubeadm).
+This guide gives an overview of how to set up NVIDIA Device Plugin which enable user to run GPUs with Yunikorn, for more details please check [**Installing the NVIDIA Container Toolkit**](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html).
 
 ### Prerequisite
-Before following the steps below, Yunikorn need to deploy on the [**Kubernetes with GPUs**](https://docs.nvidia.com/datacenter/cloud-native/kubernetes/install-k8s.html#install-kubernetes).
+Before following the steps below, Yunikorn need to deploy on the Kubernetes with GPUs.
 
 ### Install NVIDIA Device Plugin
-Add the nvidia-device-plugin helm repository.
```
Comment (craigcondit) [inline docs/user_guide/workloads/run_nvidia.md:35]: Helm installation is still supported upstream in the latest release (see the GitHub page for details). Only the version has changed (0.12.3 -> 0.14.1). We should prefer using helm repos here.

File: docs/user_guide/workloads/run_nvidia.md [hunk 2]
```diff
@@ -26,30 +26,16 @@ under the License.
 -->
 
 ## Yunikorn with NVIDIA GPUs
-This guide gives an overview of how to set up NVIDIA Device Plugin which enable user to run GPUs with Yunikorn, for more details please check [**Kubernetes with GPUs**](https://docs.nvidia.com/datacenter/cloud-native/kubernetes/install-k8s.html#option-2-installing-kubernetes-using-kubeadm).
+This guide gives an overview of how to set up NVIDIA Device Plugin which enable user to run GPUs with Yunikorn, for more details please check [**Installing the NVIDIA Container Toolkit**](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html).
```
Comment (craigcondit) [inline docs/user_guide/workloads/run_nvidia.md:29]: This is not the Kubernetes documentation -- this is for other container runtimes. Please link to the Kubernetes documentation instead.

**Review by craigcondit — CHANGES_REQUESTED** [review]
There seems to be some confusion here about what's needed. From what I can see the instructions are largely correct, however, the original Kubernetes with GPUs page has moved on the NVidia site.

File: docs/user_guide/workloads/run_nvidia.md (hunk 1 above)
Comment (KatLantyss) [reply docs/user_guide/workloads/run_nvidia.md:35]: @craigcondit Thanks for the mention and review! I didn't notice that Helm installation is still supported before, and I fix it now!

File: docs/user_guide/workloads/run_nvidia.md (hunk 2 above)
Comment (KatLantyss) [reply docs/user_guide/workloads/run_nvidia.md]: @craigcondit I changed to the correct link.

**Review by wilfred-s — APPROVED** [review]
LGTM
checked the URLs and they are the same as pointed to by the NVIDIA docs

**Pull request closed without merging**